src/types.ts

```diff
@@ -0,0 +1,191 @@
+export type EvaluationContext = { targetingKey?: string } & Record<string, string | number | boolean | Date>;
```

Comment:
Are you waiting on the context PR to be merged before adding the rest? Also, can you add JSDocs above each value to explain how they're used?

Reply:
>Are you waiting on the context PR to be merged before adding the rest? 

Yes.

>  Also, can you add JSDocs above each value to explain how they're used?

What do you mean when you say "value"? Each interface?